registry/registry.go

```diff
@@ -81,7 +81,16 @@ func (r Registry[T]) AllEntries() []Entry[T] {
 	}
 
 	return results
+}
 
+// AllEntries returns every Entry in the Registry that can be used as a Verifier
```

Comment:
The registry isn't Verifier specific, it's just a generic factory pattern. I don't think you need this function, you should be able to use the existing `AllEntries` function